internal/cmd/ini/common.go

```diff
@@ -129,16 +129,20 @@ func extValidator(framework, frameworkVersion string) survey.Validator {
 		}
 	}
 
+	return extValidator(exts)
+}
+
+func extValidator(validExts []string) survey.Validator {
 	return func(s interface{}) error {
 		val := s.(string)
 		found := false
-		for _, ext := range exts {
+		for _, ext := range validExts {
 			if strings.HasSuffix(val, ext) {
 				found = true
 			}
 		}
 		if !found {
-			return fmt.Errorf("invalid extension. must be one of the following: %s", strings.Join(exts, ", "))
+			return fmt.Errorf("invalid extension. must be one of the following: %s", strings.Join(validExts, ", "))
```

Comment:
Can be refined to 
```suggestion
			return fmt.Errorf("invalid extension. must be one of the following: %q", validExts)
```

Reply:
These are not equivalent. 

```
invalid extension. must be one of the following: .apk, .aab
```
vs.
```
invalid extension. must be one of the following: [".apk" ".aab"]
```

I think the first one looks more natural.

Reply:
Just a suggestion. It's up to you 😄